src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,81 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, ColonyClientV6, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeMotionLimits(
+      _,
+      { colonyAddress, userAddress, motionId, tokenDecimals },
+    ) {
+      try {
+        const colonyClient = (await colonyManager.getClient(
+          ClientType.ColonyClient,
+          colonyAddress,
+        )) as ColonyClientV6;
+        const votingReputationClient = await colonyClient.getExtensionClient(
+          Extension.VotingReputation,
+        );
+        const {
+          skillRep,
+          stakes,
+          domainId,
+        } = await votingReputationClient.getMotion(motionId);
+        const { skillId } = await colonyClient.getDomain(domainId);
+        const { reputationAmount } = await colonyClient.getReputation(
+          skillId,
+          userAddress,
+        );
+        // @NOTE There's no prettier compatible solution to this :(
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        // eslint-disable-next-line max-len
+        const userMinStakeFraction = await votingReputationClient.getUserMinStakeFraction();
+
+        const totalStaked = bigNumberify(stakes[1])
+          .div(bigNumberify(10).pow(tokenDecimals))
+          .toNumber();
```

Comment:
How is this total staked, when you are only calculating the stakes from the YAY side ?

Reply:
Sure, will change the name to `YAYSideTotal` or something like that

Reply:
No, but I meant don't we need to total both stakes for this to work? Both yay and nay

Reply:
Mmmm I'm not so sure about it. The reason as to why I'm calculating the total staked over the YAY side it's because of these 2 edge cases Alex mentioned to me:

> 1. When the motion is nearly totally staked, such that the remaining amount of stake is smaller than minstake, we always allow the user to stake that much, otherwise the motion would get stuck
> 2. Maxstake can only be up to the amount of stake required to get to 100% (note that this and the one above can be different depending on which side the user is staking on)

I use totalStaked to get what's the amount left to achieve the 100% from that side of the vote and limit the Slider based on that

Reply:
Ok, but convert it to a string rather then a number _(since it will be quite a large number, and JS can't handle it)_